README.md

```diff
@@ -86,7 +86,7 @@ You should be prompted to enter the _CircleCI API Token_ you generated from the
 
 
 ```
-✔ CircleCI API Token: 
+✔ CircleCI API Token (which can be created here: https://circleci.com/account/api): 
```

Comment:
This information is [already included in the README](https://github.com/CircleCI-Public/circleci-cli/pull/339/files#diff-04c6e90faac2675aa89e2176d2eec7d8R85).